database/frontend/images.json

```
@@ -82,5 +82,11 @@
     "url": "https://www.lifeofpix.com/",
     "category": "frontend",
     "subcategory": "images"
+  },
+    "name": "Gencraft",
+    "description": "The world’s most powerful AI photo and video art generation engine. What will you create?",
```

[Nitpick] I think this could be improved. What's your experience with this tool? Consider describing it from that viewpoint. 
```suggestion
    "description": "The world’s most powerful AI photo and video art generation engine. What will you create?",
```